packages/snaps-sdk/src/jsx/validation.test.tsx

```diff
@@ -235,6 +235,31 @@ describe('FieldStruct', () => {
     <Field label="foo">
       <Input name="foo" type="text" />
     </Field>,
+    <Field label="foo">
+      <Box>
+        <Text>Hello</Text>
+      </Box>
+      <Input name="foo" type="text" />
+    </Field>,
+    <Field label="foo">
+      <Input name="foo" type="text" />
+      <Box>
+        <Text>Hello</Text>
+      </Box>
+    </Field>,
+    <Field label="foo">
+      <Box>
+        <Text>Hello</Text>
+      </Box>
+      <Input name="foo" type="text" />
+      <Box>
```

Comment:
Can we test with Text in front of and behind please?

Reply:
Added requested test.